profile-specific-challenges/frontend/components/idioms.jsx

```diff
@@ -392,34 +459,60 @@ export function UnoptimizableRenderingStructure(altRecords) {
 }
 
 // RenderHooks
-
-function useRenderHook(number) {
-  return <div>{number}</div>;
-}
-
 export function RenderHookComponent() {
   const [counter, setCounter] = useState();
-  const number = useRenderHook(counter);
```

Comment:
I think this one is opinionated, is it just returning JSX that is wrong? There are a lot of libraries out there that return components (functions)